layers/meta-opentrons/recipes-robot/opentrons-module-firmware/opentrons-module-firmware.bb

```diff
@@ -10,12 +10,12 @@ TEMPERATURE_MODULE_SOURCE="temp-deck-arduino.ino.hex"
 TEMPERATURE_MODULE_URL="temperature-module/${TEMPERATURE_MODULE_VERSION}/${TEMPERATURE_MODULE_SOURCE}"
 SRC_URI += "${BASE_URL}/${TEMPERATURE_MODULE_URL};sha256sum=4e4ef00b44a320fe461265dbd0484d90247a13f405125657be8915c8ea214408"
 
-THERMOCYCLER-GEN2_MODULE_VERSION="v1.0.4"
+THERMOCYCLER-GEN2_MODULE_VERSION="v1.0.5"
 THERMOCYCLER-GEN2_MODULE_SOURCE="thermocycler-gen2@${THERMOCYCLER-GEN2_MODULE_VERSION}.bin"
 THERMOCYCLER-GEN2_MODULE_URL="thermocycler-gen2/${THERMOCYCLER-GEN2_MODULE_VERSION}/${THERMOCYCLER-GEN2_MODULE_SOURCE}"
-SRC_URI += "${BASE_URL}/${THERMOCYCLER-GEN2_MODULE_URL};sha256sum=0a6fd09e57368f746b2e14a7ae9ad473f7a4a944bde2e8769461aeb9c0d115fa"
+SRC_URI += "${BASE_URL}/${THERMOCYCLER-GEN2_MODULE_URL};sha256sum=e457eafad0c8f326d85c1a66e843f50745bfb56b0ccfe01ff580ddecb81b39bf"
 
-HEATER-SHAKER_MODULE_VERSION="v1.0.4"
+HEATER-SHAKER_MODULE_VERSION="v1.0.5"
```

Comment:
I don't think there is a heater shaker v1.0.5 build, the latest is v1.0.4.

Reply:
yeah this was a mistake